pkg/review/commit_review_status.go

```diff
@@ -488,8 +493,22 @@ func GetPullRequestsTargetingDefaultBranch(ctx context.Context, client *githubv4
 		// Only select pull requests made against the default branch.
 		for _, pr := range query.Repository.Object.Commit.AssociatedPullRequest.Nodes {
 			if pr.BaseRefName == query.Repository.DefaultBranchRef.Name {
+				// if review decision is not set (e.g. it is blank or null), then the
+				// repository does not have the 'Require pull request reviews before merging'
+				// branch protection setting is enabled.
+				// https://github.com/orgs/community/discussions/24375
```

Comment:
I'm OK with this but I wonder if we should have a different status to reflect such approvals? Just so that we know there is a nuance in these approvals.

Reply:
I like the idea of distinguishing this scenario from other 'APPROVED' and 'UNKNOWN' cases. I'm writing up a one pager so we can get consensus on exactly what the names and procedure should be.

Reply:
Sounds good - I agree this is a weird edge case. Thanks for writing something up so we can all align @lock14

Reply:
I am closing as using `reviewDecision` was deemed to un-reliable as it can be set to null post-merge of a PR.
We are instead switching to using the underlying review data to determine a commit's approval status.
See  #231 for this change.